platform/security/core/security-core-services/pom.xml

```diff
@@ -201,6 +201,10 @@
             <groupId>ddf.security.handler</groupId>
             <artifactId>security-handler-impl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-server</artifactId>
```

Comment:
✏️ This is fine for 2.26.x but we should try to avoid this when forward porting to master.